### PR TITLE
Add filter, risk and log endpoint clients

### PR DESCRIPTION
**Pull request by dawlib**

### CODE REVIEW

File: src/models/filter-result.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { RiskPolicy } from './risk-policy';
+import { Verdict } from './verdict';
+
+export type FilterResult = {
+
```
Comment (dawlib) [inline src/models/filter-result.ts:5]: let's allow for a flexible response, without the fixed structure like it's done for "authenticate" action for example

File: src/models/log-result.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { RiskPolicy } from './risk-policy';
+import { Verdict } from './verdict';
+
+export type LogResult = {
+
```
Comment (dawlib) [inline src/models/log-result.ts:5]: let's allow for a flexible response, without the fixed structure like it's done for "authenticate" action for example

File: test/api/services/api-filter.service.test.ts [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        user_id: 'user_id',
+      });
+
+      const config = new Configuration({
+        apiSecret: 'test',
+        overrideFetch: fetch,
+        logger: { info: () => {} },
+      });
+
+      const response = await APIFilterService.call(
```
Comment (dawlib) [inline test/api/services/api-filter.service.test.ts:44]: Response for filter has such attributes: 
```
{'action': 'allow', 'risk': 0.41, 'policy': {'id': ..., 'revision_id': ..., 'name': ...}, 'signals': {'datacenter_ip': {}}

```

File: test/api/services/api-risk.service.test.ts [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+      updated_at: 'today',
+    },
+    context: {
+      ip: '8.8.8.8',
+      headers: {},
+    },
+  };
+
+  describe('call', () => {
+    it('handles allow response', async () => {
```
Comment (dawlib) [inline test/api/services/api-risk.service.test.ts:31]: Response for risk has such attributes:

```
{'action': 'deny', 'risk': 0.89, 'policy': {'id': '9999bb59-0d60-42d8-8e38-7179d445de90', 'revision_id': 'bc666b75-03e3-4e9e-4444-3226130bab4d', 'name': 'invalidemail'}, 'signals': {'unapproved_device_type': {}, 'unapproved_device': {}, 'unapproved_country': {}, 'datacenter_ip': {}}, 'device': {'fingerprint': 'flEzW_S8S8eYYccT-2HrIw', 'connected_users_count': 1}, 'failover': ..., 'failover_reason': ...}
```

**Review by bartes — APPROVED** [review]

**Comment by dawlib** [comment]
LGTM.

**Pull request merged**